frontstage/views/contact_us.py

```diff
@@ -13,3 +22,41 @@
 @contact_us_bp.route("/", methods=["GET"])
 def contact_us():
     return render_template("contact-us.html")
+
+
+@contact_us_bp.route("/send-message", methods=["GET", "POST"])
+@jwt_authorization(request)
+def send_message(session) -> str:
+    secure_message_form = SecureMessagingForm(request.form)
+    errors = {}
+    if request.method == "POST":
+        secure_message_form.party_id = session.get_party_id()
+        secure_message_form.category = (
+            "TECHNICAL" if request.form.get("survey_id") == "Not survey related" else "SURVEY"
+        )
+
+        try:
+            msg_id = send_secure_message(secure_message_form)
+            thread_url = url_for("secure_message_bp.view_conversation", thread_id=msg_id) + "#latest-message"
+            flash(Markup(f"Message sent. <a href={thread_url}>View Message</a>"))
```

Comment:
It's not on the figma but on the pdf tthere is a note to update the response to:

> Your message has been sent, a request can take up to 5 working days. View [message](url)